pkgs/development/python-modules/eth-keyfile/default.nix

```diff
@@ -10,18 +11,22 @@
 
 buildPythonPackage rec {
   pname = "eth-keyfile";
-  version = "0.6.0";
-  format = "setuptools";
+  version = "0.8.0";
+  pyproject = true;
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "ethereum";
     repo = "eth-keyfile";
     rev = "v${version}";
     fetchSubmodules = true;
-    hash = "sha256-JD4bRoD9L0JXcd+bTZrq/BkWw5QGzOi1RvoyLJC77kk=";
+    hash = "sha256-797yhHuU9/lm96YKxl3SZ5IQAwDxDSYkLkiBdAHh0Uk=";
   };
 
+  nativeBuildInputs = [
+    setuptools-scm
```

Comment:
```suggestion
  build-system = [
    setuptools
```
and elsewhere

Reply:
how about now?

Reply:
No. Read carefully

Reply:
what's the difference between setuptools and setuptools-scm, I saw somewhere else was setuptools-scm.

Reply:
SCM sets the version from a git tag. Upstream decides what they use.

Reply:
thanks. I changed them all.